src/webgl/material.js

```diff
@@ -127,11 +130,25 @@ p5.prototype.loadShader = function(
  * // the 'varying's are shared between both vertex & fragment shaders
  * let varying = 'precision highp float; varying vec2 vPos;';
  *
+ * Shaders can alter the positioning of shapes drawn with them.
```

Comment:
Right now this code is inside the *code*  of the example, where it should probably just be with the text description. Maybe move this sentence up to be after line 119?